src/platform/utilities/feature-toggles/flipper-client.js

```diff
@@ -0,0 +1,90 @@
+// import environment from 'platform/utilities/environment';
+const TOGGLE_VALUES_PATH = '/toggle.json';
+const TOGGLE_POLLING_INTERVAL = 5000;
+
+let flipperClientInstance;
+
+function FlipperClient({
+  // host = environment.API_URL,
+  host = 'http://localhost:3001',
+  toggleValuesPath = TOGGLE_VALUES_PATH,
+} = {}) {
+  let _timeoutId;
+  let _pollingActive;
+  const _subscriberCallbacks = [];
+
+  const _fetchToggleValues = async function featureToggleValues() {
+    const response = await fetch(`${host}${toggleValuesPath}`);
+    const responseJson = response.ok
+      ? await response.json()
+      : new Error(
+          `Failed to fetch toggle values with status ${response.status} ${
+            response.statusText
+          }`,
+        );
+
+    if (responseJson instanceof Error) {
+      throw responseJson;
+    }
+
+    return responseJson;
+  };
+
+  const addSubscriberCallback = subscriberCallback =>
+    _subscriberCallbacks.push(subscriberCallback) - 1;
+
+  const handleToggleValuesRetrieved = toggleValues =>
+    _subscriberCallbacks.forEach(callback => callback(toggleValues));
+
+  const refreshToggleValues = async function refreshToggleValues() {
+    const { toggleValues } = await _fetchToggleValues();
+
+    handleToggleValuesRetrieved(toggleValues);
+  };
+
+  const fetchToggleValues = async function fetchTogggleValues() {
```

Comment:
This is the only function used. For now it wraps the returns just the toggleValues from the response. Might refactor this later- originally I didn't want to expose this and to force consumers to use the subscriber callback. Might still end up doing that.